src/data_morph/shapes/circles.py

```diff
@@ -60,7 +60,10 @@ def distance(self, x: Number, y: Number) -> float:
         float
             The absolute distance between this circle's edge and the point (x, y).
         """
-        return abs(self._euclidean_distance((self.cx, self.cy), (x, y)) - self.r)
+        return abs(
+            self._euclidean_distance(np.array([self.cx, self.cy]), np.array([x, y]))
```

Comment:
Should we just store `self._center = np.array([self.cx, self.cy])` to avoid making this over and over again (and also needing to extract this for the shapes that use multiple circles)?